qemu/tests/cfg/rh_ksleftests_vm.cfg

```diff
@@ -0,0 +1,11 @@
+- rh_ksleftests_vm:
+    only Linux
```

Comment:
only RHEL

Refer to the previous discussion of the kar patch, this patch is for RHEL only

Reply:
Done !